go.mod

```diff
@@ -82,12 +83,10 @@ require (
 	go.uber.org/zap v1.19.1
 	golang.org/x/crypto v0.0.0-20210915214749-c084706c2272
 	// BUG(INFRA-1801): Last version that supports go1.16 is golang.org/x/net v0.0.0-20211020060615-d418f374d309
-	golang.org/x/net v0.0.0-20211020060615-d418f374d309
-	golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
-	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f // indirect
+	golang.org/x/net v0.0.0-20220624214902-1bab6f366d9e
```

Comment:
Looks like the change in `go_repositories.bzl` was manually reverted to fix the build - great! The go.mod should be updated as well (revert in go.mod, use `gazelle_update_repos` to update `go_repositories.bzl`, instead of manually modifying the latter)

Reply:
uhm .. not sure about the manual revert and what needs to be done here :/ I just rebased, had some conflicts around the versioning, and brought back your versions and your comments. I don't think I have enough background to know what breaks/what needs to be done here, happy to do what you tell me to. (are there targets I should test/verify?)